src/components/Expenses/Expenses.tsx

```diff
@@ -4,15 +4,16 @@ import Card from "../UI/Card";
 import ExpenseFilter from "./ExpenseFilter";
 import ExpenseList from "./ExpenseList";
 import ExpensesChart from "./ExpensesChart";
+import { expenseData } from "../../App";
 
-function Expenses(props) {
-  const [filteredYear, setFilteredYear] = useState("2022");
+function Expenses(props: { item: expenseData[] }) {
+  const [filteredYear, setFilteredYear] = useState<string>("2022");
 
-  const showFilteredByYear = (selYear) => {
+  const showFilteredByYear = (selYear: string) => {
     setFilteredYear(selYear);
   };
 
-  const expenseYears = props.item.filter((expense) => {
+  const expenseYears = props.item.filter((expense: expenseData) => {
```

Comment:
Same here. There is no need to statically type the `expense` variable. But get in the habit of naming variables as their plural form, if they are an array. In this case it will be `items`